### PR TITLE
port to python 38

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545

### CODE REVIEW

**Comment by bissenbay** [comment]
@fridex @harshad16 We don't have .aicoe-ci.yaml file in this repo as well.

**Comment by bissenbay** [comment]
@harshad16 do I need to add `.aicoe-ci.yaml`?

**Comment by harshad16** [comment]
can you please add content to `.aicoe-ci.yaml`
```
check:
  - thoth-build
  - thoth-precommit
build:
  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
  build-stratergy: Source
  registry: quay.io
  registry-org: thoth-station
  registry-project: build-watcher
  registry-secret: thoth-station-thoth-pusher-secret
```
thanks @bissenbay , also rebase

**Comment by bissenbay** [comment]
> can you please add content to `.aicoe-ci.yaml`
> 
> ```
> check:
>   - thoth-build
>   - thoth-precommit
> build:
>   base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
>   build-stratergy: Source
>   registry: quay.io
>   registry-org: thoth-station
>   registry-project: build-watcher
>   registry-secret: thoth-station-thoth-pusher-secret
> ```
> 
> thanks @bissenbay , also rebase

thanks @harshad16 , https://github.com/thoth-station/build-watcher/pull/176/commits/2804b1d5adfa06f929576c7ba3639ee611d7625a

File: .thoth.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system:
       name: rhel
```
Comment (bissenbay) [inline .thoth.yaml:8]: @fridex @harshad16 Should I change it to `ubi`?

Comment (fridex) [reply .thoth.yaml:8]: ubi/rhel are actually synonyms on the recommender side so no need for a change

**Review by harshad16 — APPROVED** [review]
lgtm,
thanks for all the changes. :100:

File: .thoth.yaml (hunk 1 above)
Comment (bissenbay) [reply .thoth.yaml:8]: good, thanks

**Pull request merged**